### PR TITLE
Introduce new flavor for pr builds

### PR DESCRIPTION
**Pull request by malinajirka**
Goal of this PR is to introduce a new gradle flavor which will be used for PR builds. The main need for this change is that testers currently need to uninstall alpha/beta versions on their device before they can install an apk from a PR.

I've registered the new package in Firebase console. However, I haven't udpate the google-services.json on the CircleCI yet. There are more changes in the file than I anticipated and want to double check I'm using the correct file first.

### CODE REVIEW

File: WordPress/google-services.json-example [hunk 1]
```diff
@@ -46,6 +46,48 @@
         }
       }
     },
+    {
+          "client_info": {
+            "mobilesdk_app_id": "1:123:android:abc",
+            "android_client_info": {
+              "package_name": "org.wordpress.android.jalapeno"
```
Comment (theck13) [inline WordPress/google-services.json-example:53]:
```suggestion
              "package_name": "org.wordpress.android.prealpha"
```

The `package_name` value doesn't match the [`applicationId` value in the `build.gradle` file](https://github.com/wordpress-mobile/WordPress-Android/pull/11640/files?file-filters%5B%5D=.gradle&file-filters%5B%5D=.json-example&file-filters%5B%5D=.xml#diff-01b46d294a950c5acc9d967b2607983bR106).  We should update the `applicationId` value to `org.wordpress.android.jalapeno` in `build.gradle` ***or*** the `package_name` value to `org.wordpress.android.prealpha` in `google-services.json`.  If we change the `package_name` value, we need to make the change to the [other `package_name` instance below](https://github.com/wordpress-mobile/WordPress-Android/pull/11640/files?file-filters%5B%5D=.gradle&file-filters%5B%5D=.json-example&file-filters%5B%5D=.xml#diff-92d330fad535220493fa2f6dd96b0bd2R65).

File: WordPress/src/jalapeno/res/drawable/app_icon_background.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<vector
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:height="108dp"
+    android:width="108dp"
+    android:viewportHeight="108.0"
+    android:viewportWidth="108.0" >
+
+    <path
+        android:fillColor="#ff292f33"
```
Comment (theck13) [inline WordPress/src/jalapeno/res/drawable/app_icon_background.xml:9]:
```suggestion
        android:fillColor="#ff77b255"
```

In order to differentiate `jalapeno` and `wasabi` flavors, I would change the `jalapeno` background to `#77b255` to look like a jalapeno.

File: WordPress/src/jalapeno/res/drawable/app_icon_foreground.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+    <path
+        android:fillColor="#ff77b255"
+        android:pathData="M83,54c0-16-13-29-29-29c-1.8,0-3.6,0.2-5.3,0.5c-0.1,0.2-0.2,0.3-0.4,0.4c0,0-8,13.7,0,32 c2.2,4.9,12.4,17.9,15.8,23.2C75.1,77,83,66.4,83,54z" >
+    </path>
+
+    <path
+        android:fillColor="#fff4900c"
+        android:pathData="M48.3,25.9c0.2-0.1,0.3-0.3,0.4-0.4C35.3,28,25,39.9,25,54c0,16,13,29,29,29c3.5,0,7-0.6,10.1-1.8 c-3.4-5.4-13.6-18.4-15.8-23.2C40.3,39.6,48.3,25.9,48.3,25.9z" >
+    </path>
```
Comment (theck13) [inline WordPress/src/jalapeno/res/drawable/app_icon_foreground.xml:16]:
```suggestion
    <path
        android:fillColor="#ffdce775"
        android:pathData="M83,54c0,16-13,29-29,29S25,70,25,54s13-29,29-29S83,38,83,54z" >
    </path>
```

In order to differentiate `jalapeno` and `wasabi` flavors, I would change the `jalapeno` foreground to `#dce775` to look like a jalapeno.

File: WordPress/google-services.json-example (hunk 1 above)
Comment (malinajirka) [reply WordPress/google-services.json-example:53]: ohh, my bad. Good catch ;)!

File: WordPress/src/jalapeno/res/drawable/app_icon_background.xml (hunk 2 above)
Comment (malinajirka) [reply WordPress/src/jalapeno/res/drawable/app_icon_background.xml:9]: I added a red dot to the jalapeno icon. However, I agree that this change is even better ;).

File: WordPress/google-services.json-example [hunk 4]
```diff
@@ -46,6 +46,48 @@
...
+          "oauth_client": [
+            {
+              "client_id": "123-abc.apps.googleusercontent.com",
+              "client_type": 3
+            },
+            {
+              "client_id": "",
+              "client_type": 1,
+              "android_info": {
+                "package_name": "org.wordpress.android.jalapeno",
```
Comment (malinajirka) [inline WordPress/google-services.json-example:65]:
```suggestion
                "package_name": "org.wordpress.android.prealpha",
```

**Comment by malinajirka** [comment]
Awesome, thank you @theck13 ! 🙇 

I plan to update the google-services.json on the circle CI and register the new certificate fingerprint in the console before merging this PR. I'll merge it when all this is ready.

**Comment by malinajirka** [comment]
@loremattei I've fixed the google-services.json and update the .configure file. This PR should be ready for final review now. 🙇 Thanks

**Comment by malinajirka** [comment]
As it turned out, CircleCI uses a different debug keystore for each build :(. So the Google-login still doesn't work on the generated apks. Any idea why it's not using the default - `~/.android/debug.keystore` ?

**Comment by loremattei** [comment]
As discussed on Slack, I fixed the merge conflicts. 
I think this is ready to  go :-)

**Pull request merged**